internal/subscription/model.go

```diff
@@ -20,6 +20,9 @@ const (
 	SubscriptionTypePull = "Pull"
 )
 
+// List of subscription model
+type List []*Model
```

Comment:
Do we need a dedicated type for a collection? Can this be defined as an array where needed?

Reply:
yes.